src/lib/api/postsFetcher.ts

```diff
@@ -1,10 +1,11 @@
 export async function fetchPosts(
```

Comment:
Please improve file names, this could have been called getPosts.ts

Reply:
Acknowledged. Will rename other files as well.